tests/manage/monitoring/conftest.py

```diff
@@ -360,6 +360,31 @@ def workload_fio_storageutilization(
     pvc_size = \
         fiojob.get_storageutilization_size(target_percentage, ceph_pool_name)
 
+    # To handle use case of test_workload_rbd_cephfs_minimal which writes data
+    # to reach a small fraction of the total capacity only (eg. 5%), the test
+    # is going increase the target 2x and try again.
+    if pvc_size <= 0 and target_percentage <= 0.10:
```

Comment:
shouldn't it be "if pvc_size >= 0 ..." ?

Reply:
This is on purpose. If the small utilization target is already met (eg. there already is 6% of data stored in the cluster when the target is just 5%), the pvc_size to get to the target will be negative.